grafana/grafana
# Docs: Correlations

Documentation for #55236

Fixes #62100

**Special notes for your reviewer:**

To test:

```
cd ~/projects/grafana/docs 
make docs
```

Open http://localhost:3002/docs/grafana/latest/administration/correlations/


Please check that:
